spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/web/ServerPropertiesTests.java

```diff
@@ -315,6 +322,66 @@ public void handleError(ClientHttpResponse response) throws IOException {
 		}
 	}
 
+	@Test
+	public void jettyMaxHttpFormPostSizeMatchesDefault() throws Exception {
```

Comment:
Could refactor so that both this test and `jettyMaxHttpPostSizeMatchesDefault` re-use the common code. Thoughts?

Reply:
I think `jettyMaxHttpPostSizeMatchesDefault` can be removed now.